services/bitcoin.js

```diff
@@ -18,11 +18,11 @@ const bip68 = require('bip68');
 const ECPair = ECPairFactory(ecc);
 
 // TODO: remove bcoin
-const bcoin = require('bcoin');
+// const bcoin = require('bcoin');
```

Comment:
I noticed a bunch of stuff commented out. Will these be removed before merge?

Reply:
Should be fixed now!